parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -258,6 +258,7 @@ write_files:
             "isGateway": true,
             "ipMasq": false,
             "hairpinMode": false,
+            "promiscMode": true,
```

Comment:
@khenidak @aramase are there any side effects of this change?

Also should we do the equivalent change for Windows + kubenet (though it's not really a recommended configuration)

cc @marosset @jsturtevant